docs/using_zampy.md

```diff
@@ -13,26 +13,28 @@ This file is created under your -*user's home*-/.config directory:
 
```

Comment:
I cannot add a suggestion to the line above where it is mentioned "`This file is created under ...`". Is the file is created by zampy or the user should create it? For the latter, the documentation should change: "`You need to create the file under....`"

Reply:
The user needs to do this (for now, we can eventually add a config manager to the CLI). I have modified the docs to be more explicit.